security/unauthorized-actions.md

```diff
@@ -1,4 +1,20 @@
 ---
 description: >-
-  Desribes the process needed to fix the Unauthorized Actions message users come across 
----
\ No newline at end of file
+  Describes the process needed to fix the Unauthorized Actions message users come across 
+---
+
+# Unauthorized Actions 
+
+When the user tries to perform the actions that require **Administrator** application role, SysKit Sense will show dialog warning user that it does not have appropriate permissions. Learn more about Application Roles in SysKit Sense [here](./application-roles.md).

```

Comment:
a dialog

Reply:
that they do not? 
with it it sourns like syskit sense it the entity that does not have permissions